### PR TITLE
New package: crud-0.1.1

### PR DESCRIPTION
**Pull request by kojicomics**
[Crud](https://github.com/fosu/crud) is a tool which lets you select a region on the screen and prints information about that region to stdout.

### CODE REVIEW

File: srcpkgs/crud/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+makedepends="libX11-devel libXext-devel"
+short_desc="Prints the geometry of a selected region to stdout"
+maintainer="Rubén Santos <kojicomics@cocaine.ninja>"
+license="MIT"
+homepage="https://github.com/fosu/crud"
+distfiles="https://github.com/fosu/crud/archive/${version}.tar.gz"
+checksum=0d4061e8ec7f3abde891ceeb982b3303436a8e74047bd04b4501ed43d4797848
+
+do_install() {
+	vbin crud
```
Comment (maxice8) [inline srcpkgs/crud/template]: gnu-makefile runs make install which should install the program

Comment (kojicomics) [reply srcpkgs/crud/template]: The build failed with the [original make install](https://github.com/fosu/crud/blob/e979fca0050398f303ae3375232075e1cd46742c/Makefile#L14):
`install -m755 crud $(DESTDIR)$(PREFIX)/bin/crud`

This is the _xbps-src_ output:
```
=> crud-0.1_1: running do_install ...
install -m755 crud /destdir//crud-0.1/usr/bin/crud
install: cannot create regular file '/destdir//crud-0.1/usr/bin/crud': No such file or directory
make: *** [Makefile:14: install] Error 1
=> ERROR: crud-0.1_1: do_install: '${make_cmd} STRIP=true PREFIX=/usr DESTDIR=${DESTDIR} ${make_install_args} ${make_install_target}' exited with 2
=> ERROR:   in do_install() at common/build-style/gnu-makefile.sh:40
```
I wasn't able to fix that, so I wrote the _do_install_

Comment (maxice8) [reply srcpkgs/crud/template]: Simple, it is trying to install to a non-existent directory.

just do 

```
pre_build() {
    sed -i 's|install|install -d|' Makefile
}
```

Comment (kojicomics) [reply srcpkgs/crud/template]: Thanks, it's working now.

**Comment by maxice8** [comment]
I opened a PR upstream to fix this, after https://github.com/fosu/crud/pull/4 gets merged you can remove the workaround in the next update.

you might want to write a comment above the sed command to help remember.

**Comment by kojicomics** [comment]
Updated to 0.1.1 with the makefile fix merged

**Pull request merged**